content/hardware/05.pro-solutions/solutions-and-kits/portenta-proto-kit-ve/datasheet/datasheet.md

```diff
@@ -0,0 +1,380 @@
+
+### Sample Sketches
+
+<p style="text-align: justify;">Sample sketches for the Portenta Proto Kit VE can be found either in the “Examples” menu in the Arduino IDE or the “Portenta Proto Kit VE Documentation” section of Arduino documentation <strong>[4]</strong>. These examples include basic and advanced applications showcasing motion and environmental monitoring capabilities.</p>
+
+### Online Resources
+
+<p style="text-align: justify;">Now that you have gone through the basics of what you can do with the Portenta Proto Kit VE, you can explore the endless possibilities it provides by checking exciting projects on Arduino Project Hub <strong>[5]</strong>, the Arduino Library Reference <strong>[6]</strong> and the ACE-220 online course <strong>[7]</strong>. The Enterprise Prototyping with Portenta Proto Kit ME (ACE-220) course is a resource designed to help you master prototyping in embedded electronics and IoT. Gain hands-on experience with the kit and accelerate your journey from concept to innovation by building functional prototypes tailored for industrial and IoT applications.</p>
+
+<div style="page-break-after: always;"></div>
+
+## Mechanical Information
+
+<p style="text-align: justify;">
+The Arduino Portenta Proto Kit VE offers significant mechanical flexibility, supporting multiple configurations based on the combination of components used. This section provides the main dimensions of one possible configuration for reference. For detailed mechanical specifications of each individual component, please consult the corresponding documentation available on Arduino Docs.
+</p>
+
+### Kit Dimensions
+
+<p style="text-align: justify;">
+The figures below show the main dimensions of the kit in a stacked configuration that includes the Portenta H7 board, the Portenta Mid Carrier, the Portenta Mid Carrier Proto Shield, the Arduino Pro 4G Module, one Modulino (Pixels) and the Nicla Vision board. While the Nicla Sense Env is part of the kit, it is not physically stacked with these components. Instead, it connects to the system via an ESLOV cable, whose length may vary depending on the user's setup. As a result, the Nicla Sense Env and its connection are not represented in this stacked configuration. All dimensions are in millimeters (mm).
+</p>
+
+- <p style="text-align: justify;"><strong>Top View</strong>: Displays the width and length of the stacked components configuration of the kit.</p>
+
+![](assets/proto-kit-example-mechanical-1.png)
+
+<div style="page-break-after: always;"></div>
+
+- <p style="text-align: justify;"><strong>Side View</strong>: Displays the height of the stacked components configuration of the kit.</p>
+
+![](assets/proto-kit-example-mechanical-2.png)
```

Comment:
Since it explained previously: "While the Nicla Sense Env is part of the kit, it is not physically stacked with these components. Instead, it connects to the system via an ESLOV cable, whose length may vary depending on the user's setup. As a result, the Nicla Sense Env and its connection are not represented in this stacked configuration."

If possible, it would be nice to represent the height of the stacked kit until highest element (barrel from what I see) and add rest of the dimension to also show if Nicla is physically stacked at some point when applicable, thus summing up to 38.38mm . The update if applied, would represent two dimensions, instead of one for the height as currently present.

Reply:
Image updated!